Doc/whatsnew/3.10.rst

```diff
@@ -1358,3 +1358,19 @@ Removed
   AST object (``mod_ty`` type) with the public C API. The function was already
   excluded from the limited C API (:pep:`384`).
   (Contributed by Victor Stinner in :issue:`43244`.)
+
+* Remove the ``symtable.h`` header file and the undocumented functions:
```

Comment:
Not sure if this should be here, as it was never documented

Reply:
Previously, I didn't document such changes, but there is always one project sleeping somewhere which relies on the modified/removed function, and then it's painful to understand why the project is broken. Even if a function is very well hidden and not documented, there is always a secret project relying on it :-)